opba-rest-api-shared/src/main/java/de/adorsys/opba/restapi/shared/service/FacadeResponseMapper.java

```diff
@@ -93,4 +103,28 @@
         result.getHeaders().forEach(builder::header);
         return builder;
     }
+
+    private ResponseEntity.BodyBuilder setCookieHeaders(FacadeResultRedirectable<?, ?> result, ResponseEntity.BodyBuilder builder) {
+        if (Strings.isNullOrEmpty(result.getToken())) {
+            return builder;
+        }
+
+        String redirectPath = fromAspspRedirectPath(result.getAuthorizationSessionId(), result.getRedirectCode());
+        ResponseCookie responseCookie = cookieBuilderTemplate.builder(result.getToken(),
+                                                                     redirectPath,
+                                                                     cookieProperties.getRedirectMaxAge())
+                                               .build();
+
+        builder.header(COOKIE_TTL, Long.toString(cookieProperties.getRedirectMaxAge().getSeconds()))
+                .header(SET_COOKIE, responseCookie.toString());
+
+        return builder;
+    }
+
+    public String fromAspspRedirectPath(String authorizationId, String redirectState) {
+        return UriComponentsBuilder.fromPath("/embedded-server/v1/consent/{authorizationSessionId}/fromAspsp/{redirectState}/*")
```

Comment:
'/embedded-server/v1/consent/{authorizationSessionId}/fromAspsp/{redirectState}/*'
should come from configuration properties

Reply:
done